### ISSUE TITLE
Add existing item as Link -> Compiler cannot find @import file

### ISSUE THREAD

**Issue by LemonStore** [failwyn/WebCompiler#21]
### Installed product versions
- Visual Studio: 2022
- This extension: 1.14.2

### Description
When I have a less files in another location and "add an item as link", Webcompiler cannot find file

### Steps to recreate
1.
(P1) ASP.NET Core 6.0 Project (Microsoft.NET.Sdk.Web)
(P2) Class Library (Microsoft.NET.Sdk.Razor)

2.
Create a "colors.less" file in P2
Create a "site.less" file in P1

3.
In P1, add existing file and select the ""colors.less" from P2 and in the dropdown, "Add As Link"
Add **@import "Colors.less";** to "site.less"

4.
(Re)Compile "site.less"

### Current behavior
FileError: 'colors.less' wasn't found.

### Expected behavior
Should compile "site.less"
[css.zip](https://github.com/failwyn/WebCompiler/files/8009870/css.zip)

**Comment by failwyn** [comment]
Do you know if this worked in the VS2019 version?

**Comment by LemonStore** [comment]
Probably not, I uninstalled 2019 a while ago.

**Comment by failwyn** [comment]
I’ll take a look to see what it would take to support linked files; are you good to work around this for now?

**Comment by LemonStore** [comment]
Yes, I just make a copy of the file and sync it manually. Thanks for your fast reaction.

**Comment by failwyn** [comment]
It doesn't look like this is going to be possible since the files are compiled using NodeJs and it doesn't look at the project file to see where the actual file is located; it also doesn't look like LESS has an option similar to LoadPaths in SASS.

**Comment by gh-at-sqh** [comment]
Reading other issues, I thought we could direct node using at least the "options" parameter for SASS files like this:

`"options": { "loadPaths": "../../../../../Web/Assets/scss/" }`

But, alas, no joy either.

How do I find out if the node routines are looking at all, or looking in the right place? Error is just "Can't find stylesheet to import." Is there a debug output of the process somewhere?
